packages/react-scripts/package.json

```diff
@@ -60,6 +60,7 @@
     "postcss-loader": "0.13.0",
     "promise": "7.1.1",
     "react-dev-utils": "^0.2.1",
+    "react-hot-loader": "^3.0.0-beta.6",
```

Comment:
It seems to be working rather well. I don't think there's a problem with "production-readiness" because this isn't put in the production build. 

My thought process behind using the v3 instead of the current release is I'm betting on less changing from the beta to the final than from the current to the final of v3. Does that make sense?

If we use this and it starts to cause issues, we will also be able to release a fix to this package only and tell folks to update.

WDYT?